clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -386,14 +386,23 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
   }
 
   mlir::cir::VectorType getExtendedElementVectorType(mlir::cir::VectorType vt,
-                                                     bool isSigned = false) {
+                                                     bool isSigned) {
     auto elementTy =
         mlir::dyn_cast_or_null<mlir::cir::IntType>(vt.getEltType());
     assert(elementTy && "expected int vector");
     return mlir::cir::VectorType::get(
         getContext(), getExtendedIntTy(elementTy, isSigned), vt.getSize());
   }
 
+  mlir::cir::VectorType getExtendedElementVectorType(mlir::cir::VectorType vt) {
+    auto elementTy =
+        mlir::dyn_cast_or_null<mlir::cir::IntType>(vt.getEltType());
+    assert(elementTy && "expected int vector");
+    bool isSigned = elementTy.isSigned();
+    return mlir::cir::VectorType::get(
+        getContext(), getExtendedIntTy(elementTy, isSigned), vt.getSize());
+  }
```

Comment:
I'm having a hard time to understand why we basically have two copies of the same function here... please fix this.

Reply:
It was introduced on purpose. In one situation, it is used to inherit the signess from the source element type. In another, the client specifies the sign. They are for different purposes. But I'm open to better ideas here.

Reply:
Now, I'm thinking, if leave the signess query to the client side, we don't need another helper. just fixed it that way.